clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -4082,6 +4082,8 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     CmdArgs.push_back(Args.MakeArgString(NormalizedTriple));
   }
 
+  Arg *SYCLStdArg = Args.getLastArg(options::OPT_sycl_std_EQ);
+
```

Comment:
I'm not sure I understand the purpose of `UseSYCLTriple` flag, but driver should ignore `-sycl-std` option if SYCL mode is not enabled (i.e. no `-fsycl` option).

Reply:
I can't really see the point of your message.

Reply:
Sorry for the noise. I just misinterpret the code.
But anyway, is there any test checking that `-sycl-std` driver option is ignored in non-SYCL mode?

Reply:
@s-kanaev, I believe that to address this, `test/Driver/sycl.c:15-22` could be extended with a test case like:
```
// RUN: %clangxx -### -sycl-std=2017 -- %s 2>&1 | FileCheck %s --check-prefix=DISABLED
// RUN: %clang_cl -### -sycl-std=2017 -- %s 2>&1 | FileCheck %s --check-prefix=DISABLED
```
The necessary `DISABLED` line is already in place (line 22):
`// DISABLED-NOT: "-sycl-std="`
and it may be needed to rewrite it like:
`// DISABLED-NOT: "-sycl-std={{[0-9]+}}"`

Reply:
Done